mapping/charging.py

```diff
@@ -107,3 +107,32 @@ def map_charging_bna(row, station_id):
     charging.max_kw = max(kw_list) if kw_list else None
     #charging = _clean_attributes(charging)
     return charging
+
+
+def map_charging_ocm(row, station_id):
+    # TODO: compute kW if missing and possible
+    capacity: int = row["NumberOfPoints"]
+    kw_list: List[float] = []
+    ampere_list: List[float] = []
+    volt_list: List[float] = []
+    socket_type_list: List[str] = []
+
+    # Stations with only Schuko-Steckern are no charging stations for cars.
+    #if kw_list and max(kw_list) < MIN_KW:
```

Comment:
why is this commented, do we filter out charging stations with very low KW later or not at all?

Reply:
i will check that